mod/05_standard/set/self/account_links.rb

```diff
@@ -5,8 +5,9 @@
     #ENGLISH
     links = []
     if Auth.signed_in?
-      links << link_to_page( Auth.current.name, nil, :id=>'my-card-link' )
+      links << build_link( Auth.current.cardname, nil, :id=>'my-card-link' )
```

Comment:
I think this stuff works.  I don't recall as I comment, but I think this is just a matter of accepting options to be used as attributes of the link element.